browser/ui/webui/wallet_panel/wallet_panel_page_handler.h

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2021 The Brave Authors. All rights reserved.
```

Comment:
these should go in `webui/brave_wallet` to match the component

Reply:
or webui/brave_wallet/wallet_panel, but in general we should "namespace" things by the component and then if you need subdirs from there

Reply:
Fixed in ede26f8912e32e574ff3bead0d2facbfc74fc8f5